### ISSUE TITLE
Enterprise SSO Registration at topcoder.com Signup Error 

### ISSUE THREAD

**Issue by lakshmiathreya** [topcoder-platform/tc-auth-lib#147]
-- Enterprise SSO signup is expected to be via the enterprise site - for example - for Wipro Enterprise it would be topgear.topcoder.com.
-- However, if an Enterprise User happens to attempt signup via topcoder.com, the Signup button should either redirect to the appropriate Enterprise site or show a relevant message mentioning the Enterprise site.
-- Currently - on macOS, Windows and Android, the Signup button doesnt work. It does work as expected on iOS. 
<img width="1680" alt="Screenshot 2021-06-23 at 3 25 52 PM" src="https://user-images.githubusercontent.com/51733042/123083747-03b22700-d43e-11eb-817e-9c7c8bc7e4c7.png">

**Comment by lakshmiathreya** [comment]
<img width="1680" alt="Screenshot 2021-06-23 at 3 17 37 PM" src="https://user-images.githubusercontent.com/51733042/123084379-abc7f000-d43e-11eb-8daa-607c36c56aa5.png">

**Comment by lakshmiathreya** [comment]
This error is observed on macOS, Android and Windows. The Enterprise SSO signup flow works without Signup error on iOS.
cc @drasticdpk

**Comment by lakshmiathreya** [comment]
This error is observed on macOS, Android and Windows. The Enterprise SSO signup flow works without Signup error on iOS.
cc @drasticdpk

**Comment by Oanh-and-only-Oanh** [comment]
> -- Enterprise SSO signup is expected to be via the enterprise site - for example - for Wipro Enterprise it would be topgear.topcoder.com.
> -- However, if an Enterprise User happens to attempt signup via topcoder.com, the Signup button should either redirect to the appropriate Enterprise site or show a relevant message mentioning the Enterprise site.

Correct, this is the expected flow and should be working on macOS, Android, and Windows.

**Comment by sachin-maheshwari** [comment]
@lakshmiathreya can you please share the recordings for all OS/browsers where you faced the issue.

**Comment by drasticdpk** [comment]
@sachin-maheshwari , 
here is the reference 

**Windows/Chrome**

https://user-images.githubusercontent.com/25297859/123130873-e052a080-d46c-11eb-8a78-42916982e169.mp4


**MAC/Chrome**
https://user-images.githubusercontent.com/25297859/123131143-23ad0f00-d46d-11eb-99e3-695da97a091f.mp4


**Android**
https://user-images.githubusercontent.com/25297859/123132453-7b984580-d46e-11eb-82d4-8333c40ec0bd.mp4

**Finding** 

If user type ****@wipro.com** in userName/handle filed, it is redirecting to SSO page, please confirm

![image](https://user-images.githubusercontent.com/25297859/123133220-3d4f5600-d46f-11eb-80ce-ff8e56ea4a66.png)

**Comment by lakshmiathreya** [comment]
@Oanh-and-only-Oanh - what is the expected landing page for Enterprise users? In the iOS scenario - it successfully redirects after signup to the respective Member/Customer landing pages. Should we redirect them to Topgear instead?

**Comment by lakshmiathreya** [comment]
https://user-images.githubusercontent.com/51733042/123210569-cdc57f00-d4df-11eb-85bd-b7fd0881e19e.MP4

**Comment by lakshmiathreya** [comment]
https://user-images.githubusercontent.com/51733042/123211106-9c997e80-d4e0-11eb-9d12-a5140fbbfc9a.mp4

**Comment by lakshmiathreya** [comment]
<img width="1680" alt="Screenshot 2021-06-24 at 11 47 27 AM" src="https://user-images.githubusercontent.com/51733042/123212163-10885680-d4e2-11eb-9d3b-f61c2ec51b2d.png">

**Comment by lakshmiathreya** [comment]
<img width="1680" alt="Screenshot 2021-06-24 at 11 47 27 AM" src="https://user-images.githubusercontent.com/51733042/123212163-10885680-d4e2-11eb-9d3b-f61c2ec51b2d.png">

**Comment by Oanh-and-only-Oanh** [comment]
> @Oanh-and-only-Oanh - what is the expected landing page for Enterprise users? In the iOS scenario - it successfully redirects after signup to the respective Member/Customer landing pages. Should we redirect them to Topgear instead?

@lakshmiathreya, If user type **@wipro.com in userName/handle filed, it should redirect to to SSO page, 
![image](https://user-images.githubusercontent.com/47536078/124784040-673a5b00-defa-11eb-93c3-cb36ce330786.png)
